polly/plugins/publish/validate_deadline_done.py

```diff
@@ -26,15 +26,20 @@ def process(self, instance):
             6: "Pending",
         }
 
-        url = api.Session["AVALON_DEADLINE"] + "/api/jobs?JobID=%s"
+        assert "AVALON_DEADLINE" in api.Session, ("Environment variable "
```

Comment:
Nitpick, but I think this is a little clearer.

```python
assert "AVALON_DEADLINE" in api.Session, (
  "Environment variable missing from current session: AVALON_DEADLINE"
)
```

Also the word "session" isn't something the end-user is aware of, to him, it's just an environment variable that's gone missing.